tools/perf_checker/test_anndata_export.py

```diff
@@ -0,0 +1,19 @@
+from sys import stderr
+
+import cellxgene_census
+import tiledbsoma as soma
+
+print("Starting bm 1", file=stderr)
+census_S3_latest = dict(census_version="2024-01-01")
```

Comment:
```suggestion
census_S3_latest = dict(census_version="latest")
```

Reply:
The issue with using the latest is that it may lead to performance degradation once the latest changes

Reply:
If you pin a non LTS version it will go away after one month. That said, this is a test file so it's probably fine, but leave a comment for posterity.